site2/docs/io-influxdb-sink.md

```diff
@@ -0,0 +1,61 @@
+---
+id: io-influxdb-sink
+title: InfluxDB sink connector
+sidebar_label: InfluxDB sink connector
+---
+
+The InfluxDB sink connector pulls messages from Pulsar topics 
+and persists the messages to InfluxDB.
+
+## Configuration
+
+The configuration of the InfluxDB sink connector has the following properties.
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `influxdbUrl` |String| true|" " (empty string) | The URL of the InfluxDB instance. |
+| `username` | String|false| " " (empty string) |The username used to authenticate to InfluxDB. |
+| `password` | String| true|" " (empty string)  | The password used to authenticate to InfluxDB. |
```

Comment:
The field `password` is optional

Reply:
Thank you, I've corrected. Could you PTAL?